integration_tests/load_tests.feature

```diff
@@ -3,7 +3,7 @@ Feature: Load Tests
   Scenario: DefinedLoadStrategy follows the profile provided by the user as dict
     Given we have a scenario named strategy_tests/user_profile_load_dict
     And d3a is installed
-    When we run the d3a simulation with strategy_tests.user_profile_load_dict [24, 15, 15]
+    When we run the d3a simulation with strategy_tests.user_profile_load_dict [None, 24, 15, 15]
```

Comment:
Well, you already made this change therefore no point in reverting it, but you could have created a separate behave step implementation that would automatically match the [d, h, m, s] format, and would redirect to the existing function. Anyways, since you already made the changes it's ok.

Reply:
OMG! You are right. Did not think about that. If you think this looks ugly and should rather be changed, I am happy to add another step. Please let me know. Thanks

Reply:
Actually, considering the change to @faizan2590 s comment, I will implement e separate step for multi-day simulations and get rid of the `None`s all over the place. :-)